testing/test_aux.out

```diff
@@ -100,11 +103,15 @@ port-02 module_config modules
   "brute": {
     "enabled": true
   },
+  "dhcp": {
+    "timeout_sec": 120
+  },
   "hold": {
     "enabled": true 
   },
   "macoui": {
-    "enabled": true
+    "enabled": true,
+    "timeout_sec": 1
```

Comment:
looks like the macoui tests takes about ~1.3 seconds (just looking at the logs) -- so might be some flakyness here if some system decided to be fast.  If it works it works, but just something to watch out for.